Sources/ToyboxKit/Handler.swift

```diff
@@ -57,11 +57,14 @@ public struct XcodeOpener: PlaygroundOpenerType {
 }
 
 public struct PlaygroundHandler<Workspace: WorkspaceType, Loader: TemplateLoaderType, Opener: PlaygroundOpenerType> {
+    private let autoremoveSuffix = ".autoremove"
+
     public var rootURL: URL {
         return Workspace.rootURL
     }
 
     public init() {
+        executeAutoremove()
```

Comment:
This is in the initializer because I want to remove files every command execution. Do you have any better idea?